File: backend/log/log.go
package log

import (
	"fmt"
	"time"
)

func Warning(t string) {
	timestamp := time.Now().Format("2006-01-02 15:04:05")
	fmt.Println(timestamp + " | " + "[WARNING]" + " | " + "backend/api/api.go" + " | " + t)
}

func Info(t string) {
	timestamp := time.Now().Format("2006-01-02 15:04:05")
	fmt.Println(timestamp + " | " + "[INFO]" + " | " + "backend/api/api.go" + " | " + t)
}

func Fatal(t string) {
	timestamp := time.Now().Format("2006-01-02 15:04:05")
	fmt.Println(timestamp + " | " + "[FATAL]" + " | " + "backend/api/api.go" + " | " + t)
}

File: backend/flask-server/log.py
from datetime import datetime

class Log:

    def __init__(self, name):
        self.name = name

    def warning(self, text):
        print(f"{datetime.now()} | [WARNING] | {self.name} | {text}\n")

    def fatal(self, text):
        print(f"{datetime.now()} | [FATAL] | {self.name} | {text}\n")

    def info(self, text):
        print(f"{datetime.now()} | [INFO] | {self.name} | {text}\n")